pkgs/tools/archivers/cpio/default.nix

```diff
@@ -2,34 +2,13 @@
 
 stdenv.mkDerivation rec {
   pname = "cpio";
-  version = "2.13";
+  version = "2.14";
 
   src = fetchurl {
     url = "mirror://gnu/cpio/cpio-${version}.tar.bz2";
-    sha256 = "0vbgnhkawdllgnkdn6zn1f56fczwk0518krakz2qbwhxmv2vvdga";
+    sha256 = "/NwV1g9yZ6b8fvzWudt7bIlmxPL7u5ZMJNQTNv0/LBI=";
   };
 
-  patches = let
```

Comment:
`fetchpatch` is no more used in the derivation, it can be removed L1.

Reply:
Done,